packet/utils.py

```diff
@@ -85,3 +85,10 @@ def wrapped_function(*args, **kwargs):
         return func(*args, **kwargs)
 
     return wrapped_function
+
+def notify_slack(name: str):
+    """
+    Sends a congratulate on sight decree to Slack.
+    """
+    msg = f'{name} got :100: on packet. Congratulate on sight.'
```

Comment:
If you don't want the source to say "Shower on sight", maybe pull the message from an env variable?
I'm fine with it saying shower on sight personally